pyaedt/edb_core/nets.py

```diff
@@ -291,5 +291,6 @@ def is_net_in_component(self, component_name, net_name):
         if component_name not in self.parent.core_components.components:
             return False
         for net in self.parent.core_components.components[component_name].nets:
-            return net_name == net.GetName():
+            if net_name == net.GetName():
+                return True
```

Comment:
@akaszynski that's change is not correct. In my code I want to return Only True when the match is achieved. in your case you return also in case of False

Reply:
This still returns `True` when the two variables match and `False` when they do not match, but without the extra branch:

Here's a basic example:

```py

def return_match_long(var_a, var_b):
    if var_a == var_b:
        return True
    return False

def return_match_short(var_a, var_b):
    return var_a == var_b


>>> return_match_long('foo', 'foo')
True
>>> return_match_long('foo', 'bar')
False

>>> return_match_short('foo', 'foo')
True
>>> return_match_short('foo', 'bar')
False
```

These two are identical, but the `return_match_short` is less complex.

Reply:
Nevermind, I see what you did there.  Sorry!